index-flex1.html

```diff
@@ -8,26 +8,29 @@
     <title>Lambda - Home</title>
 
     <link href="https://fonts.googleapis.com/css?family=Roboto|Rubik" rel="stylesheet">
-    <link rel="stylesheet" href="css/index.css">
+    <link rel="stylesheet" href="./css/index-flex1.css">
 
 </head>
 
+
 <div class="missing-container">
+    <header class="top-row">
     <img class="logo"
-        src="/Users/jasminejavid/Lambda/Sprint-Challenges/Sprint-Challenge--User-Interface/img/lambda-black.png"
+        src="./img/lambda-black.png"
         alt="black lambda logo">
-    <header class="top-row">
+    
 
         <nav class="missing-nav">
             <a class="link1" href="#">Home</a>
-            <a class="link2" href="about.html">About</a>
+            <a class="link2" href="./about-flex1.html">About</a>
             <a class="link3" href="#">Products</a>
             <a class="link4" href="#">Blog</a>
             <a class="link5" href="#">Contact</a>
         </nav>
+    </div>
```

Comment:
what does the closing div tag belong to? I would investigate if this should be there.